dts/xtensa/espressif/esp32.dtsi

```diff
@@ -273,6 +274,17 @@
 			status = "disabled";
 		};
 
+		twai: can@3ff6b000 {
+			compatible = "espressif,esp32-twai";
+			reg = <0x3ff6b000 DT_SIZE_K(4)>;
+			interrupts = <TWAI_INTR_SOURCE>;
+			interrupt-parent = <&intc>;
+			clocks = <&rtc ESP32_TWAI_MODULE>;
+			sjw = <1>;
+			sample-point = <875>;
```

Comment:
Is this a flexible parameter? The sample point according to CAN bus specification should reside between 60 to 70% of bit rate period and depends on SJW and the CAN BUS time quanta 1 and 2, what would happen here if other bit rate got selected this value will be the same? Same for the sjw, it would assume values between 1 and 3 time quanta.

Reply:
According to CAN in Automation (CiA) the sample point shall reside at 87.5%. This is just a default setting, though. Both the sample point and the SJW can be changed at runtime using the CAN timing APIs.